pkg/cloudprovider/providers/aws/tags_test.go

```diff
@@ -108,3 +114,68 @@ func TestFindClusterID(t *testing.T) {
 		}
 	}
 }
+
+func TestHasClusterTag(t *testing.T) {
+	awsServices := NewFakeAWSServices(TestClusterId)
+	c, err := newAWSCloud(CloudConfig{}, awsServices)
+	if err != nil {
+		t.Errorf("Error building aws cloud: %v", err)
+		return
+	}
+	grid := []struct {
+		Tags     map[string]string
+		Expected bool
+	}{
+		{
+			Tags: map[string]string{},
+		},
```

Comment:
Nit: might be clearer to put `Expected: false` here even though it's the default - just to make it more obvious